frontend/src/pages/login.js

```diff
@@ -1,177 +1,14 @@
-import React, { useState, useEffect, useContext } from "react";
```

Comment:
Why are we deleting the OAuth functionality?

Reply:
Because we only need to use a personal token for the GitHub API. At least that is the easiest option,